### ISSUE TITLE
A11y: Add favicon to eCR Viewer

### ISSUE THREAD

**Issue by angelathe** [CDCgov/dibbs-ecr-viewer#1156]
## What needs to be done

Add (back?) the favicon to the eCR Viewer. Note: Please consult design about what favicon should be used.

Example:
<img width="554" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeXAxQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--59b48ac8bfd21fdd3455c1583725c11c28ab7c16/Screenshot%202025-09-19%20at%2019.09.48.png" alt="Screenshot 2025-09-19 at 19.09.48.png" />

## Why it needs to be done

This follows the accessibility audit scans for the eCR Viewer pages. (This is not a necessary a11y issue, but I thought we did this before -- don't know what happened it)

## Acceptance Criteria

- [ ] The DIBBs eCR Viewer page has a favicon

## Technical Notes

Axe Issue: N/A (was not an issue, just something discussed)

This is low priority
